libs/shared/db/mysql/account/src/lib/sent-email.ts

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { BaseModel } from './base';
+
+/**
+ * Track subscription related emails sent to FxA users.
+ */
+export class SentEmail extends BaseModel {
```

Comment:
I like how clean this model is. What's the plan for the static methods that are on the class currently fxa-shared. e.g. https://github.com/mozilla/fxa/blob/2b7a7044dfbcb01132d63c3553e629d54225ad3a/packages/fxa-shared/db/models/auth/sent-email.ts#L52